Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -134,6 +138,21 @@ void systemInit(void)
     //
     configurePWM();
 
+    //
+    // Copy time critical code and Flash setup code to RAM
+    // This includes the following ISR functions: epwm1_timer_isr(),
+    // epwm2_timer_isr(), epwm3_timer_isr and and InitFlash();
+    // The  RamfuncsLoadStart, RamfuncsLoadSize, and RamfuncsRunStart
+    // symbols are created by the linker. Refer to the F2808.cmd file.
+    //
+    //memcpy(&RamfuncsRunStart, &RamfuncsLoadStart, (Uint32)&RamfuncsLoadSize);
+
+    //
+    // Call Flash Initialization to setup flash waitstates
+    // This function must reside in RAM
+    //
+    //InitFlash();
+
```

Comment:
Why including this comments?

Reply:
You will have to remove the comment when you want to flash the DSP.
A detailed description of what else to do is added to the trello card:
https://trello.com/c/XliY0e1j/116-watchdog-flash-programming